src/useStyles.js

```diff
@@ -0,0 +1,54 @@
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = (borderStyle) => makeStyles((theme) => {
+  const containerBorderStyle = (borderStyle.size && borderStyle.type) && {
```

Comment:
we can probably cut out the conditional statement as both `size` and `type` will contain values because of `propType`